integration/src/test/scala/ammonite/integration/StandaloneTests.scala

```diff
@@ -124,6 +124,83 @@ object StandaloneTests extends TestSuite{
       assert(e.toString.contains(expectedErrorMsg))
     }
 
+    'compilationErrorLineNumberTest1{
+      //Make sure correct line numbers are printed when an erroneous script is executed
+      val name = "compilationErrorLineNumberTest1.scala"
+      val e = intercept[ShelloutException]{
+        %%bash(
+          executable,
+          "--predef-file",
+          emptyPrefdef,
+          replStandaloneResources / name
+          )
+      }
+      val expectedErrorMsg =
+        """Main.scala:11: not found: value quicort
+          |    quicort(unsorted.filter(_ < pivot)):::List(pivot):::""".stripMargin +
+          """quicksort(unsorted.filter(_ > pivot))"""
+
+      assert(e.toString.contains(expectedErrorMsg))
+
+    }
+
+    'compilationErrorLineNumberTest2{
+      //Make sure correct line numbers are printed when an erroneous script is executed
+      val name = "compilationErrorLineNumberTest2.scala"
+      val e = intercept[ShelloutException]{
+        %%bash(
+          executable,
+          "--predef-file",
+          emptyPrefdef,
+          replStandaloneResources / name
+          )
+      }
+      val expectedErrorMsg =
+        """Main.scala:14: not found: value printnl
```

Comment:
Could you change the file-name down below where you update the positions so it has the actual file name instead of `Main.scala`?